.circleci/config.yml

```diff
@@ -25,14 +25,10 @@ jobs:
       - run:
           name: Gradle Tasks
           command: |
-            ./gradlew build dependencyCheckAnalyze sonarqube \
-              -Dsonar.projectKey=$CIRCLE_PROJECT_REPONAME \
-              -Dsonar.organization=cx \
-              -Dsonar.host.url=https://sonarcloud.io \
-              -Dsonar.pullrequest.key=${CIRCLE_PULL_REQUEST##*/} \
-              -Dsonar.pullrequest.branch=$CIRCLE_BRANCH \
-              -Dsonar.pullrequest.base=master \
-              -Dsonar.login=$SONAR_TOKEN \
+            if [[ ! -z $CIRCLE_PULL_REQUEST ]];
```

Comment:
❓What does this `if` statement check?

Reply:
This checks that the environment variable `CIRCLE_PULL_REQUEST` is not empty. This is necessary because to get the PR number, we need to parse the PR URL and get the last bit after the slash (the PR number). You can see that here: `-Dsonar.pullrequest.key=${CIRCLE_PULL_REQUEST##*/}`